portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -251,27 +253,69 @@ function allowUserSubmit() {
         }
       });
 }
-google.charts.load('current', {'packages': ['corechart']});
-google.charts.setOnLoadCallback(drawStateIncarcerationChart);
-
-/** Creates chart and adds it to the page. */
-function drawStateIncarcerationChart() {
-  const stateData = new google.visualization.DataTable();
-  stateData.addColumn('string', 'Race');
-  stateData.addColumn('number', 'Count');
-  stateData.addRows([
-    ['Whites', 201],
-    ['Blacks', 1767],
-    ['Hispanics', 385],
-  ]);
-
-  const stateChartDimensions = {
-    'title': 'Incarceration Rates (per 100,000) by Race in California',
-    'width': 500,
-    'height': 400,
+
+drawTimeSeriesChart();
+
+/** Adds a line chart to page showing the global avg temp from a csv */
+async function drawTimeSeriesChart() {
+  const tempData = await getTempData();
+  const chart = {
+    type: 'line',
+    data: {
+      labels: tempData.xAxis,
+      datasets: [{
+        label: 'Mean Temp',
+        backgroundColor: 'rgb(255, 0, 0)',
+        fill: false,
+        borderColor: 'rgb(255, 0, 0)',
+        data: tempData.yAxis,
+      }],
+    },
+
+    options: {
+      reponsive: true,
+      title: {
+        display: true,
+        text: 'Zonal Annual Temperature Mean',
+      },
+      scales: {
+        xAxes: [{
+          display: true,
+          scaleLabel: {
+            display: true,
+            labelString: 'Year',
+          },
+        }],
+        yAxes: [{
+          display: true,
+          scaleLabel: {
+            display: true,
+            labelString: 'Temperature Mean',
+          },
+        }],
+      },
+    },
   };
```

Comment:
I think this semicolon is placed incorrectly. This seems to be the end of your function so that semicolon should be placed at the previous curly-bracket.

Reply:
The bracket on line 297 is the end of the options section while the bracket on line 298 is the end of the function. Placing the semicolon on the previous bracket would cause an error